EntAppProject/src/main/resources/static/css/JobBookLoginUtil.css

```diff
@@ -0,0 +1,2993 @@
+/*[ FONT SIZE ]
```

Comment:
This and most of the other files that have been created in the static folder (unless they have a comment) are necessary files that have been added as a styling foundation and to support the updated Login page. If the team decides to keep these changes, but does not want the application to have so many more resources, the team can remove all animations files easily without breaking anything. However, I was able to use the animations on my computer easily, and in 2019 most computers can handle the minimal animations that these files provide. Thus, in my opinion the benefit outweighs the cost.

The greatest benefit of using these files is that it includes a local copy of bootstrap as well as an example of how to use it in an HTML page (JBLogin.html). I recommend examining that example thoroughly and applying the bootstrap classes to the other HTML pages because it will greatly improve the visuals of the application without much work.